src/interface/index.ts

```diff
@@ -0,0 +1,24 @@
+export interface SignupData {
```

Comment:
make separate file for putting signUp interface. Do put all the interface of the entire application in a single file. Categorise it and make separate file according that.

Reply:
Also follow a naming pattern for interface, Ex: SignupDataInterface,SignupResponseInterface